packages/ckeditor5-dev-release-tools/lib/tasks/preparerepository.js

```diff
@@ -0,0 +1,115 @@
+/**
+ * @license Copyright (c) 2003-2023, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+const fs = require( 'fs-extra' );
+const upath = require( 'upath' );
+
+/**
```

Comment:
Please write a few words about the purpose of this function.

I wonder if it shouldn't be called `prepareRepositoryToRelease()` as `prepareRepository()` does not explain what it prepares. It is placed in the `release-tools` package, though. I am just thinking.